lib/websocket/driver/hybi/message.rb

```diff
@@ -14,7 +14,7 @@ def initialize
           @rsv2   = false
           @rsv3   = false
           @opcode = nil
-          @data   = Driver.encode('', :binary)
```

Comment:
@miukki I'm not sure this is related to this diff, but if you're seeing this error please open a new issue with an example demonstrating the bug happening.

For what it's worth: the 'opcode' is a piece of metadata that's part of all WebSocket frames that tells the parser which type of frame it is: text data, binary data, a continuation of a previous frame, a closing message, a ping or a pong. I'm not sure how a WebSocket parser could consider a frame to have an opcode of -1, since the opcode is encoded in the protocol as an unsigned 4-bit integer.